modules/system/assets/js/framework.js

```diff
@@ -423,6 +424,7 @@ if (window.jQuery.request !== undefined) {
             flash: $this.data('request-flash'),
             files: $this.data('request-files'),
             form: $this.data('request-form'),
+            location: $this.data('request-url'),
```

Comment:
Why is the internal option `location` and not `url`?

Reply:
Fixed.